src/backend/utils/indexer.js

```diff
@@ -52,7 +52,6 @@ const deletePost = async (postId) => {
  */
 const search = async (textToSearch) => {
   const query = {
-    _source: ['id'],
```

Comment:
Does this fix what we discussed in the other PR that Stella is working on?  If so, can you mention that in the other PR?

Reply:
I pushed a new commit to remove the above line so she can take care of it in her issue